data/records/delphi-conditions-database.json

```diff
@@ -0,0 +1,75 @@
+[
+  {
+    "abstract": {
+      "description": "The DELPHI conditions database."
```

Comment:
The record looks currently like:

![image](https://github.com/user-attachments/assets/5c969b74-1005-4d89-8149-415bd2e462ed)

The abstract is a bit short, so casual visitors may be perplexed.

Can you please extend the text and perhaps add some usage information and links to docs etc? You could for example explain what `DBcalb.dat` means, what `DBsysf.dat` means, etc.

Example from CMS: https://opendata.cern.ch/record/1818 with the sources in `data/records/cms-condition-data-2016.json`.